src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
-body {
+@use './card';
+@use './stars';
```

Comment:
Consider importing a separate SCSS file for variables to ensure that all main values are consistent and easily maintainable across different style files.